.circleci/config.yml

```diff
@@ -0,0 +1,244 @@
+# Default configuration file for Drupal modules.
+#
+# Use setup.sh to automate setting this up. Otherwise, to use this in a new
+# module:
+#   1. Copy config.yml to the module's .circleci directory.
+#   2. Change 'latest' in the image tag to the latest tag.
+#   3. Update the working_directory key.
+#   4. Connect CircleCI to the repository through the Circle UI.
+#   5. Set the COMPOSER_AUTH environment variable in Circle to grant access to
+#      any private repositories.
+#   6. Create a status badge embed code in Circle and add it to the README.md.
+#
+# Check https://circleci.com/docs/2.0/language-php/ for more details
+#
+
+defaults: &defaults
+  docker:
+    # specify the version you desire here (avoid latest except for testing)
+    - image: andrewberry/drupal_tests:0.4.0
```

Comment:
This is using `drupal:8.6-apache` which is php 7.2 do we want to have the ability to test against multiple versions of PHP?